assets/javascripts/modules/analytics/omniture.js

```diff
@@ -66,7 +60,7 @@ define([
     function onSuccess() {
         window.s_account = 'guardiangu-subscribe,guardiangu-network';
         s = s_gi('guardiangu-network');
-        if(!document.querySelector('main[data-no-page-load-tracking]')) {
+        if (!document.querySelector('main[data-no-page-load-tracking]')) {
```

Comment:
Mentioned in the omniture PR but could we define whether we enable this in the global config, rather than via an attr on main?